### PR TITLE
feat: group @types/jest and jest monorepo

### PR DESCRIPTION
**Pull request by F3n67u**
## Changes:

group `@types/jest` and jest monorepo

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/config/presets/internal/group.ts [hunk 1]
```diff
@@ -596,6 +597,15 @@ const staticGroups = {
       },
     ],
   },
+  jestPlusTypes: {
+    description: 'Add @types/jest update to Jest monorepo',
+    packageRules: [
+      {
+        matchPackageNames: '@types/jest',
```
Comment (rarkins) [inline lib/config/presets/internal/group.ts:604]:
```suggestion
        matchPackageNames: ['@types/jest'],
```

Comment (F3n67u) [reply lib/config/presets/internal/group.ts]: Thanks for your review @rarkins

**Comment by F3n67u** [comment]
@rarkins  should i remove this line also? https://github.com/renovatebot/renovate/blob/f075ffc0170e416df6d64249ddff8e9105025119/renovate.json#L26

**Comment by rarkins** [comment]
> @rarkins should i remove those line also?
> 
> https://github.com/renovatebot/renovate/blob/f075ffc0170e416df6d64249ddff8e9105025119/renovate.json#L25-L28

No, not until the new version of Renovate is live in the app

**Review by rarkins — APPROVED** [review]

**Pull request merged**